saltproc/tests/test_sparger.py

```diff
@@ -12,13 +12,13 @@
 
 serpent = DepcodeSerpent(
     exec_path='/home/andrei2/serpent/serpent2/src_2131/sss2',
```

Comment:
Consider replacing with 'sss2', assuming user has run update-alternatives on serpent. Alternatively, could replace with variable defined in centralized location asking for user's sss2 path.

Reply:
This is a good catch, but in practice we don't call serpent here since it's just a unit test. I'm going to be redoing the way we declare these kinds of variables in #155, including removing any hardcoded paths in the test files.